### PR TITLE
Add expand-single-section attribute to <amp-accordion>. 

### LINKED ISSUE #11359: [FR] amp-accordion 'collapse others' option
Need to be able to trigger collapsing of other expanded menu items when expanding a new item in the accordion.  For example, if the accordion contains items [A (expanded), B (collapsed), C (collapsed)],  when B is clicked on, the accordion's new state should be [A (collapsed), B (expanded), C (collapsed)].  Also if C is clicked, the new state should be [A (collapsed), B (collapsed), C (expanded)]

This is very important to improve UX by preventing any long menus from spilling over multiple screens when explored on.

Ideally, an additional flag such as "amp-accordion collapse-others" should help.

### PR DESCRIPTION
**Pull request by nainar**
This PR does the following
- Adds an `expand-one` attribute to `<amp-accordion>` that ensures that only one `<section>` in `<amp-accordion>` is expanded at any given time. 
- Adds a test for the expansion/collapse of `<section>`s in <amp-accordion>
- Adds an example file (I found it weird that `<amp-accordion>` didn't have an example page. Is that for a reason?). This example needs some work. 

Fixes #11359

### CODE REVIEW

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 1]
```diff
@@ -224,9 +224,20 @@ class AmpAccordion extends AMP.BaseElement {
       } else {
         toExpand = !section.hasAttribute('expanded');
       }
+
       if (toExpand) {
         section.setAttribute('expanded', '');
         header.setAttribute('aria-expanded', 'true');
+        // if expand-one is set, only allow one <section> to be expanded at a time
+        if (this.element.hasAttribute('expand-one')) {
+          this.sections_ = this.getRealChildren();
```
Comment (cvializ) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Is it necessary to reassign `this.sections_`? Will it be different than the `this.sections_` assigned in `buildCallback()`?

File: extensions/amp-accordion/amp-accordion.md [hunk 2]
```diff
@@ -69,10 +69,13 @@ To see more demos of the `amp-accordion`, visit [AMP By Example](https://ampbyex
 {% endcall %}
 
 ### Actions
+
 #### `toggle`
 This action toggles between the `expanded` and `collapsed` states of the `amp-accordion`. When called with no arguements, it will toggle all sections of the accordion. A single section may be specified with the `section` arguement and the corresponding `id` as the value.
+
```
Comment (cvializ) [inline extensions/amp-accordion/amp-accordion.md]: Nice, the spacing looks better 👍

File: extensions/amp-accordion/0.1/test/test-amp-accordion.js [hunk 3]
```diff
@@ -105,6 +105,38 @@ describes.realWin('amp-accordion', {
...
 
+  it('should collapse other sections when expand action is triggered on a ' +
+    'collapsed section if expand-once attribute is set',
+  () => {
+    return getAmpAccordion().then(ampAccordion => {
+      ampAccordion.setAttribute('expand-one', '');
+      expect(ampAccordion.hasAttribute('expand-one')).to.be.true;
+      const impl = ampAccordion.implementation_;
+      const headerElements = doc.querySelectorAll(
+          'section > *:first-child');
```
[the remark this answers is not in the document]
Comment (nainar) [reply extensions/amp-accordion/0.1/test/test-amp-accordion.js]: Yup, thanks for bringing this to my attention

File: examples/accordion.amp.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="./regular-html-version.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+</head>
+<body>
+  <amp-accordion expand-one>
```
[the remark this answers is not in the document]
Comment (aghassemi) [reply examples/accordion.amp.html]: sgtm, descriptive.

[the remark this answers is not in the document]
Comment (nainar) [reply examples/accordion.amp.html]: Change made in latest commit.

**Comment by nainar** [comment]
@cvializ / @aghassemi for review

File: extensions/amp-accordion/amp-accordion.md [hunk 5]
```diff
@@ -86,6 +89,10 @@ Set this attribute on the `<amp-accordion>` to opt out of preserving the collaps
 
 Set this attribute on a `<section>` to display the section as expanded on page load.
 
+#####  expand-single-section
```
Comment (aghassemi) [inline extensions/amp-accordion/amp-accordion.md]: please add ticks ` around the attribute name

Comment (nainar) [reply extensions/amp-accordion/amp-accordion.md]: Done and did this for the other 2 attributes as well.

**Comment by nainar** [comment]
Hi @cvializ could you merge if this looks good to you? Thanks!

**Comment by cvializ** [comment]
Looks good, thanks @nainar!

**Pull request merged**